include/rocksdb/db.h

```diff
@@ -611,6 +611,12 @@ class DB {
     return GetIntProperty(DefaultColumnFamily(), property, value);
   }
 
+  // Reset internal stats for DB and all column families.
+  // Note this doesn't reset options.statistics.
+  virtual Status ResetInternalStats() {
```

Comment:
By the way, clearing options.statistics in this function is not easy to do. DB doesn't own options.statistics, and it can be shared across multiple DBs. We can create a Statistics.Reset() for users to call but it's inappropriate for DB to reset the object that it doesn't own.